src/main/java/it/pagopa/interop/probing/eservice/registry/reader/config/PropertiesLoader.java

```diff
@@ -36,29 +35,15 @@ public class PropertiesLoader {
 	private static PropertiesLoader instance;
 
 	/** The props. */
-	private Properties props;
-
-	/** The Constant PROPERTIES. */
-	public static final String PROPERTIES = "application.properties";
+	private Map<String, String> props;
 
 	/**
 	 * Load properties.
 	 *
 	 * @return the properties
-	 * @throws IOException Signals that an I/O exception has occurred.
 	 */
-	public PropertiesLoader() throws IOException {
-		InputStream inputStream = PropertiesLoader.class.getClassLoader().getResourceAsStream(PROPERTIES);
-		this.props = new Properties();
-		try {
-			this.props.load(inputStream);
-		} catch (IOException e) {
-			log.error("Error during reading properties from file");
-			throw e;
-		} catch (NullPointerException e) {
-			log.error("Error during reading properties from file");
-			throw e;
-		}
+	public PropertiesLoader() {
+		this.props = System.getenv();
```

Comment:
isn't it possible to keep the `application.properties` file?
taking values blindly directly from env makes really difficult to understand which configurations are required for the application

Reply:
In a java main application we do not have a container that injects the variables into the secrets, so we decided to set the variables via kubernetes into the env. Maybe we can group all the keys used to take the variables into a single file, in order to be more clear.

Reply:
setting kubernetes secrets into application env variables is ok, and it should be similar to the previous approach, where the `PropertiesLoader` loaded env vars based on `application.properties`.
Or am I missing something?